moto/cloudfront/models.py

```diff
@@ -274,6 +278,13 @@ def get_distribution(self, distribution_id: str) -> Tuple[Distribution, str]:
         dist.advance()
         return dist, dist.etag
 
+    def get_distribution_config(self, distribution_id: str) -> Tuple[Distribution, str]:
+        if distribution_id not in self.distributions:
+            raise NoSuchDistribution
```

Comment:
Ideally this gets a separate test case, as it is distinct behaviour that we want to verify

Reply:
What do you mean? I've included a test case for this new `get_distribution_config`

Reply:
I meant the behaviour of calling the method with an unknown/missing identifier.

See `test_create_distribution_with_mismatched_originid` as an example - it verifies that Moto returns the correct error message when incorrect parameters are supplied. We should have a similar test case that verifies what happens when calling `get_distribution_config(Id="unknown")`

Reply:
Totally agree, I'll review it. Thanks!

Reply:
Included test case @bblommers.